scripts/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Base image
+FROM ubuntu
+
+# Update the sources list
+RUN apt-get update
+
+# Install packages
+RUN apt-get install -y tar git curl nano wget dialog net-tools build-essential python python-dev python-distribute python-pip zlib1g-dev python-virtualenv apache2 libapache2-mod-wsgi
+
+# Enable wsgi module
+RUN a2enmod wsgi
+
+# Create cache directories
+RUN mkdir /var/cache/apache2/python-egg-cache && \
+    chown www-data:www-data /var/cache/apache2/python-egg-cache/
+
+# Set up GA4GH server
+RUN mkdir /srv/ga4gh
+WORKDIR /srv/ga4gh
+RUN virtualenv ga4gh-server-env
+RUN /bin/bash -c "source ga4gh-server-env/bin/activate"
+RUN pip install git+https://github.com/ga4gh/server
```

Comment:
The current version PyPI server has a wormtable dependency that won't
compile because of a missing header,

On Thu, Apr 2, 2015 at 1:02 AM Jerome Kelleher <notifications@github.com>
wrote:

> In scripts/Dockerfile
> <https://github.com/ga4gh/server/pull/326#discussion_r27638512>:
>
> > +# Install packages
> > +RUN apt-get install -y tar git curl nano wget dialog net-tools build-essential python python-dev python-distribute python-pip zlib1g-dev python-virtualenv apache2 libapache2-mod-wsgi
> > +
> > +# Enable wsgi module
> > +RUN a2enmod wsgi
> > +
> > +# Create cache directories
> > +RUN mkdir /var/cache/apache2/python-egg-cache && \
> > +    chown www-data:www-data /var/cache/apache2/python-egg-cache/
> > +
> > +# Set up GA4GH server
> > +RUN mkdir /srv/ga4gh
> > +WORKDIR /srv/ga4gh
> > +RUN virtualenv ga4gh-server-env
> > +RUN /bin/bash -c "source ga4gh-server-env/bin/activate"
> > +RUN pip install git+https://github.com/ga4gh/server
>
> Should we install from git or from PyPI for the default Docker install? I
> would say PyPI probably.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/server/pull/326/files#r27638512>.
>